code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -107,7 +107,7 @@
 	ignore_flags = 1 //so you can medipen through hardsuits
 	reagent_flags = DRAWABLE
 	flags_1 = null
-	list_reagents = list(/datum/reagent/medicine/epinephrine = 10, /datum/reagent/toxin/formaldehyde = 3)
+	list_reagents = list(/datum/reagent/medicine/epinephrine = 10, /datum/reagent/toxin/formaldehyde = 3, /datum/reagent/medicine/coagulent = 1)

```

Comment:
you need to increase the maximum chem capacity and the amount of chems injected per jab to compensate for the added chem here

Reply:
done